### PR TITLE
Migrate fids frontend components to new location

### PR DESCRIPTION
**Pull request by Dogrock**
The frontend part of fids gets it's own repo now over at https://github.com/flightaware/fids_frontend

The remaining backend component has been tidied up to based on this change. 
Some additional changes are made to the docker compose to make sure this new dedicated service is properly pulled in and started with the rest of the Firestarter applications.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -106,23 +106,22 @@ services:
...
+      internal:
+        aliases:
+          - fids-backend
     environment:
       # URL to database that is being updated by db-updater.
       # Documentation at https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls
       - FLIGHTS_DB_URL=${FLIGHTS_DB_URL:-sqlite:///db/flights.db}
       - POSITIONS_DB_URL=${POSITIONS_DB_URL:-postgresql://postgres:positions@timescaledb:5432}
-      - GOOGLE_MAPS_API_KEY
+      - GOOGLE_MAPS_API_KEY=${GOOGLE_MAPS_API_KEY:-}
```
Comment (NasaGeek) [inline docker-compose.yml:124]: Does this change the behavior in some way?

File: fids/app.py [hunk 2]
```diff
@@ -37,25 +37,22 @@
...
 
 UTC = timezone.utc
 
 
-# Let frontend handle any unknown routes
+# Return a 404 for any unknown routes
 @app.route("/", defaults={"path": ""})
 @app.route("/<path:path>")
 def catch_all(path):
     """Render HTML"""
```
Comment (NasaGeek) [inline fids/app.py:51]: We're not rendering any HTML anymore

Comment (NasaGeek) [reply fids/app.py:51]: Hm, what happens if we remove this altogether? Would flask default to a 404 for unknown routes in the first place?

File: fids/app.py [hunk 3]
```diff
@@ -233,4 +230,9 @@ def airport_scheduled(airport: str) -> Response:
     return jsonify([dict(e) for e in result])
 
 
+@app.route("/api/mapskey")
+def map_proxy() -> Response:
```
Comment (NasaGeek) [inline fids/app.py:234]: Function name could be clearer since this isn't proxying anything

File: fids/app.py (hunk 2 above)
Comment (Dogrock) [reply fids/app.py:51]: Yep - 404 is a default response so this route is now redundant. Removing this.

File: docker-compose.yml (hunk 1 above)
Comment (Dogrock) [reply docker-compose.yml:124]: This will default the the value to an empty string instead of being unset if not present in the .env.

**Comment by Dogrock** [comment]
Note CI will fail until https://github.com/flightaware/fids_frontend/pull/1 is merged. The CI full integration test is is looking for a container that that will be built by the GH Action introduced in said pull

**Review by NasaGeek — APPROVED** [review]

**Pull request merged**